### ISSUE TITLE
Wrong code example in Recipes -> Development Environment -> Javascript Usage

### ISSUE THREAD

**Issue by NayanJD** [Urigo/graphql-modules#1271]
The code provided to inject constructor argument in vanilla JS with Inject decorator in the docs is not working. 

In docs:

```
require('reflect-metadata');
Inject(AProvider)(MyProvider, undefined, 0); // 
Inject(ModuleConfig)(MyProvider, undefined, 0); // The third argument should be 1
module.exports = Injectable({...})(MyProvider);
```

**Comment by ardatan** [comment]
@NayanJD Could you create a reproduction using CodeSandbox?

**Comment by NayanJD** [comment]
Here you go -> https://codesandbox.io/s/node-js-forked-wlg4s?file=/src/index.js

Change third argument in line no 47 from 0 to 1 to see the difference in terminal output.

**Comment by kamilkisiela** [comment]
We released a new major version (v1.0) of GraphQL Modules. Here's the [migration guide](https://graphql-modules.com/docs/next/recipes/migration).

The new version doesn't solves the issue (it will in one of next minor releases) but since the old version becomes legacy, I'm closing the thread.

Please try to upgrade. If you can't and still want to use the legacy version, please open a new issue.
